File: scanner/actions/ScriptAction.js
var exec = require('child_process').exec
const Action = require('./Action')

class ScriptAction extends Action {
  process() {
    const envVars = this.envVars()
    const scriptPath = __dirname + '/../../config/' + this.config.filename
    exec(scriptPath, {env: envVars}, function(error, stdout, stderr) {
      console.log(stdout, stderr, error)
    })
  }
}

module.exports = ScriptAction

File: scanner/actions/script_action.py
import os
import subprocess

from .action import Action

CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))


class ScriptAction(Action):
    def process(self):
        env_vars = self.env_vars()
        script_path = CURRENT_DIR + "/../../config/" + self.config["filename"]

        subprocess.check_call(script_path.split(" "), env=env_vars)
